pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -879,6 +891,17 @@ func (m *migrationMonitor) startMonitor() {
 	}
 }
 
+// logMigrationInfo logs the same migration info as `virsh -r domjobinfo`
+func logMigrationInfo(logger *log.FilteredLogger, uid string, info *libvirt.DomainJobInfo) {
+	logger.Info(fmt.Sprintf(`Migration info for %s: TimeElapsed:%dms DataProcessed:%dB DataRemaining:%dB DataTotal:%dB `+
```

Comment:
What about log verbosity?

Reply:
Yeah I don't know.. Any suggestion?

Reply:
What if we will make it V(4) by default, but we will update the verbosity level dynamically if we will sense a hint of a problematic migration?
I can think of a number of such hints:
 - migration is taking too long already. we can start logging when we are > acceptableCompletionTime * 10%
 - we see that we are making no progress but less than the proressTimeout
 - dirty rate is high ... not sure what level is considered high, but maybe relative to MemBps ?

Reply:
I'd be for keeping it simple, I'd leave it as it is in the PR since we don't log so much in virt-launcher and this is precious info.

Reply:
I would be for the simple `V(4)` for now and improvements can be made later(maybe printing information once we autodetect slow convergence or so...).